lib/babel.js

```diff
@@ -24,11 +23,9 @@ var powerAssert = createEspowerPlugin(babel, {
 	patterns: require('./enhance-assert').PATTERNS
 });
 
-// if generators are not supported, use regenerator
 var options = {
-	blacklist: hasGenerator ? ['regenerator'] : [],
-	optional: hasGenerator ? ['asyncToGenerator', 'runtime'] : ['runtime'],
```

Comment:
Oh, it still would be if we want to avoid the unnecessary transform. I think maybe `asyncToGenerator` was rolled in a preset? There's certainly a way around it.